### PR TITLE
Multiple process parameters interface

### PR DESCRIPTION
**Pull request by josenavas**
Updates the interface to allow multiple processing parameters. The interface is based in the parameter set name and shows the values to the user.

@antgonza this should be the last needed bit to support ITS.

### CODE REVIEW

**Comment by antgonza** [comment]
Errors!! Could you fix?

File: qiita_db/parameters.py [hunk 1]
```diff
@@ -66,6 +66,56 @@ def create(cls, param_set_name, **kwargs):
...
+        """The values of the parameter set
+
+        Returns
+        -------
+        dict
+            Dictionary with the parameter values keyed by parameter name
+        """
+        conn_handler = SQLConnectionHandler()
+        result = dict(conn_handler.execute_fetchone(
+            "SELECT * FROM qiita.{0} WHERE {1} = %s".format(
```
Comment (antgonza) [inline qiita_db/parameters.py]: Will it be easier to simply retrieve the info you need vs. having to delete the ones that you do not need?

File: qiita_pet/templates/study_description_templates/preprocessed_data_info_tab.html [hunk 2]
```diff
@@ -1,5 +1,19 @@
 {% from qiita_pet.handlers.util import download_link_or_path %}
 
+<script>
+function parameters_changed(obj, ppd_id){
+  data = "foo"
```
Comment (antgonza) [inline qiita_pet/templates/study_description_templates/preprocessed_data_info_tab.html]: Do you really need to initialize this variable? If so, does it have to be "foo" or can it be simply ""?

**Comment by antgonza** [comment]
Works great but some comments on the code.

File: qiita_db/parameters.py (hunk 1 above)
Comment (josenavas) [inline qiita_db/parameters.py]: This method belongs to a base class, so I don't know which are those columns. However, I know that the two columns that are below exist, and they are not needed, so I just remove them.

Comment (antgonza) [inline qiita_db/parameters.py]: Got it, now why do you remove them vs. leaving? I guess they are always there, right?

File: qiita_pet/templates/study_description_templates/preprocessed_data_info_tab.html (hunk 2 above)
Comment (josenavas) [inline qiita_pet/templates/study_description_templates/preprocessed_data_info_tab.html]: Fixed

File: qiita_db/parameters.py (hunk 1 above)
Comment (josenavas) [inline qiita_db/parameters.py]: Those values are not technically part of the parameters. They're id's and name's used for internal use, and we can access them using properties

Comment (antgonza) [inline qiita_db/parameters.py]: OK thanks.

File: qiita_pet/util.py [hunk 3]
```diff
@@ -50,3 +50,21 @@ def clean_str(item):
...
+    ----------
+    param : BaseParameters
+        The parameter to generate the str
+
+    Returns
+    -------
+    str
+        The html string with the parameter set values
+    """
+    result = ["<b>%s:</b> %s" % (name, value)
```
Comment (ElDeveloper) [inline qiita_pet/util.py]: IMHO using a table would make more sense in this case, not sure that this should block this PR though.

File: qiita_db/parameters.py [hunk 4]
```diff
@@ -66,6 +66,56 @@ def create(cls, param_set_name, **kwargs):
 
         return cls(id_)
 
+    @classmethod
+    def iter(cls):
```
Comment (ElDeveloper) [inline qiita_db/parameters.py]: Should this be `__iter__` instead?

**Pull request merged**